js/app/improve-this-page.js

```diff
@@ -8,7 +8,7 @@ $(document).ready(function() {
     }
 
     var feedbackMessage = (
-        '<span id="feedback-form-confirmation" class="font-size--16">Thank you. Your feedback will help us as we continue to improve the service.</span>'
+        '<span id="feedback-form-confirmation" class="font-size--18">Thank you. Your feedback will help us as we continue to improve the service.</span>'
```

Comment:
Maybe worth setting `aria-live="polite"` so that screen readers will read this text when it's added to the DOM?

Reply:
Good idea :)